cache/lru_cache.cc

```diff
@@ -466,7 +467,7 @@ Cache::Handle* LRUCacheShard::Lookup(
       e->Ref();
 
       if (wait) {
```

Comment:
```wait``` is passed by the user, and may be false even for the compressed sec cache. It just means the caller, i.e BlockBasedTable, may or may not want this call to be synchronous. If ```wait``` is false, then the call to Promote() will happen in WaitAll(). In that case, we still need to set or unset the is_in_sec_cache flag. I suggest instead of calling SetIsInSecondaryCache() in Promote(), we do it here based on what's returned by Lookup().

Reply:
Updated accordingly.